controller/model/base_model.go

```diff
@@ -32,8 +32,15 @@ type BaseModelEntity interface {
 	GetCreatedAt() time.Time
 	GetUpdatedAt() time.Time
 	GetTags() map[string]interface{}
+}
+
+type BoltEntitySink interface {
```

Comment:
are we leaking persistence implementation details?

Reply:
The model needs to know how to convert to/from bolt types. However, I should probably make these types package private, since nothing outside the model package needs them.

Reply:
Fixed